bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/LgwReadingConverter.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.jeelink.internal.lacrosse;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.openhab.binding.jeelink.internal.JeeLinkReadingConverter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Converter for converting a line read from a LGW to a LgwReading.
+ *
+ * @author Volker Bier - Initial contribution
+ */
+public class LgwReadingConverter implements JeeLinkReadingConverter<LgwReading> {
```

Comment:
Please add `@NonNullByDefault`

Reply:
LgwReadingConverter.java:[35,1239]: Illegal redefinition of parameter inputLine, inherited method from JeeLinkReadingConverter<LgwReading> does not constrain this parameter